docs/api/creating-api.md

```diff
@@ -0,0 +1,115 @@
+# Creating the API
+
+## Creating the API on the OneBlink Console
+
+Firstly, we will need to make sure we have something to deploy to when we are ready. Let us go onto the [OneBlink Console](console.oneblink.io). Once we are in our account, go to the "API Hosting" side tab under the Advanced heading. Once here, let us click the plus button in the bottom right hand corner.
+
+![An image showing the API Hosting Page highlighting the button that needs to be clicked"](../pics/APIHostingPointingArrow.png)
+
+It's time to fill out some information. We only need to fill in the API subdomain, so call the subdomain what you would like. In my example, I will call it "example".
+
+![An image showing the API Hosting Page with the create an API modal](../pics/APIHostingSavingExample.png)
+
+Now you should see it featured with no environments as shown below. Now, we can get onto the fun part!
+
+## Create a Folder and call it whatever you want
+
+This folder will be the central spot for all of your API work. In this folder, we are going to create a few important files to start off with. The first file will we will create is the `.blinkmrc.json` file. This stores all the important information regarding your routes, api name, AWS Lambda configuration and much more. We won't overload you with all the different choices you have in it yet, but first we will create this file and you can copy this template over!
+
+```json
+{
+  "server": {
+    "project": "your-api-name-here.api.oneblink.io",
+    "routes": [
+      {
+        "route": "",
+        "module": ""
+      }
+    ],
+    "cors": true
+  }
+}
+```
+
+This will be updated later on in the guide. But for now, please change the project name to what you made your api name in the OneBlink Console.
+
+Next, in the folder, create another folder called "src" then, create a folder within "src" called "routes". Shown below:
+
+Now, if we are using VSCode, open a terminal with <code>crtl+shift+`</code>. If not, you will need to open your terminal and manually direct it to the root of the folder you have just created. Now you need to write
+
+```bash
+  oneblink api scope accountName-apiSubDomain.api.oneblink.io
+```
+
+You will need to replace accountName with your accountName and replace the apiSubDomain with the Sub Domain you wrote. In this case, mine would be `josh-example.api.oneblink.io`.
+
+After this, we can start writing some code, we are free to either write in `JavaScript` or `TypeScript`. We will be writing all of our tutorials in JavaScript as JavaScript does not require anything to be installed through npm.
```

Comment:
Might be better to omit this, might cause more confusion than helps for a beginner